keps/sig-node/2400-node-swap/README.md

```diff
@@ -622,12 +637,21 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+KubeletConfiguration has set `failOnSwap: false`.
```

Comment:
Can I tell two nodes apart:
- one has `failOnSwap: false` and `memorySwap` set to `swapBehavior: LimitedSwap`, with the `NodeSwap` feature gate enabled
- another has `failOnSwap: false` and the `NodeSwap` feature gate disabled

via the kubernetes API? If so, I'd mention how to distinguish them.

Reply:
I'm not sure if that is bubbled up to the API Server. The purpose of this question is for beta, when the feature gate is defaulted on, so you can't rely on it being turned on as a sign that the feature is in use. We might be able to check if `swapBehavior` is explicitly set, but empty string is equivalent to LimitedSwap.

Realistically, this KEP iterates on the existed unsupported configuration with `failOnSwap: false`. Because it was previously unsupported, I am assuming here that a production environment would not have it set if it were not using this feature.

Reply:
Once this is beta we should assume that people have this feature gate set to a value of their choice. For alpha it was different: you need to be a little more brave to try it and most clusters run with the default, ie feature enabled.

The switch from unsupported to “mostly supported, but it's still beta” is why I'm asking about observability.